internal/server/server.go

```diff
@@ -73,6 +75,16 @@ type UIOptions struct {
 	FS fs.FS `config:"-"`
 }
 
+type TLSOptions struct {
+	// CA is a PEM encoded public certificate for the CA that signed the
```

Comment:
nit: this comment implies that the CA should be a public authority which should not be required.
```suggestion
	// CA is a PEM encoded certificate for the CA that signed the
```